### ISSUE TITLE
Great plugin, doesn't like "relative extrusion" though

### ISSUE THREAD

**Issue by ntoff** [iraytrace/BlenderGcodeImport#1]
If gcode is imported from a slicer such at skeinforge and the extrude mode is set to relative, the path is broken up and doesn't display properly. 

Blender 2.73

**Comment by iraytrace** [comment]
Yes, I use Slic3r and Solidify3D so I didn't have examples of relative motion.  Hack it in and contribute back if you can.
